plugins/automation/admin/assets/javascripts/admin/components/automation-list.gjs

```diff
@@ -213,7 +215,7 @@ export default class AutomationList extends Component {
                   <DButton
                     @icon="trash-can"
```

Comment:
```suggestion
                    @isLoading={{this.isDestroying}}
                    @icon="trash-can"
```

Minor UX change but it'll be nice to display a loading state on the button when the request is being processed.